zomes/rea_economic_resource/storage/src/lib.rs

```diff
@@ -236,6 +244,12 @@ impl Updateable<EventCreateRequest> for EntryData {
             } else { self.current_location.to_owned() },
             contained_in: self.contained_in.to_owned(),
             note: self.note.to_owned(),
+            primary_accountable: if e.to_resource_inventoried_as.to_owned().is_some() && e.get_action() == "raise" {
+                Some(e.receiver.to_owned())
+            }
+            else {
+                self.primary_accountable.to_owned()
+            },
```

Comment:
this is the other logic in question. I know for sure that it doesn't use `raise` like you have here, but the more general question is how/when should this actually be updated, if at all. Working on it.

Reply:
@weswalla since we (at least I) don't understand this yet, let's just change this to say
```
primary_accountable: self.primary_accountable.to_owned(),
```
meaning it can't be changed. We can maybe separate the scope of work for doing 'transfers' of economic resoruces into a new scope, since it still seems ambiguous.